Makefile

```
@@ -0,0 +1,14 @@
+# Makefile for edx-lint
+
+.PHONY: default test
+
+default: test
+
+test:
+	python test/test_func.py
+
+clean:
+	-rm -rf *.egg-info
+	-rm -f *.pyc */*.pyc */*/*.pyc */*/*/*.pyc */*/*/*/*.pyc */*/*/*/*/*.pyc
+	-rm -rf __pycache__ */__pycache__ */*/__pycache__ */*/*/__pycache__ */*/*/*/__pycache__ */*/*/*/*/__pycache__
+	-rm -f MANIFEST
```

Ignore errors: https://www.gnu.org/software/make/manual/html_node/Errors.html#Errors